### PR TITLE
Crypten Merge to Master

### PR DESCRIPTION
**Pull request by gmuraru**
## Description
Merge the CrypTen branch into Master
**THIS WILL DROP SUPPORT FOR PYTHON3.6**

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
 - Unit tests for CrypTen

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: pip-dep/requirements.txt [hunk 1]
```diff
@@ -1,7 +1,9 @@
 openmined.threepio==0.2.0
 aiortc==0.9.28
+dill
```
Comment (LaRiffle) [inline pip-dep/requirements.txt:3]: can we add a version?

File: pip-dep/requirements.txt [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
 flask_socketio~=4.2.1
 Flask~=1.1.1
+git+https://github.com/facebookresearch/CrypTen.git#egg=crypten
```
Comment (LaRiffle) [inline pip-dep/requirements.txt:6]: Can you fix a commit maybe?

Comment (gmuraru) [reply pip-dep/requirements.txt:6]: Added

Comment (LaRiffle) [reply pip-dep/requirements.txt:6]: Sorry I meant should we add a commit number for the crypten master branch?

Comment (gmuraru) [reply pip-dep/requirements.txt:6]: Ahh..sorry - meant to comment to the dill

**Pull request merged**